web/utils/auth.js

```diff
@@ -16,18 +15,29 @@ const sessionStore = new MySQLStore({
   }
 }, pool)
 
-const auth = nextConnect() 
-    .use(session({
+const auth = session({
             name: "sid",
             secret: process.env.SESSION_SECRET || 'keyboard cat',
-            resave: false, // no need to store the session again everytime for every user
+            resave: false,
             saveUninitialized: false,
             cookie: {
                 httpOnly: true,
-                secure: process.env.NODE_ENV === 'production',
-                maxAge: 300 // 5 mins, might want to add to serverconstants.js later
+                secure: false,
+                maxAge: 300 
             },
             store: sessionStore
-        }))
+            })
 
-export default auth
\ No newline at end of file
+const getUserById = (id) => {
```

Comment:
(Not sure why my previous comment went onto a commit in your repo, but reposting it here)

We'll probably need a lot of functions like this in the future. We should probably make new util classes for user functions, profile functions, job functions, etc. What do you think?

Reply:
Yes I agree, I just moved this to utils because it was becoming quite redundant having a lot of these folders such as 'middleware' 'utils' 'config'.. We should probably seperate server-side utils and frontend ones to make things more concise. Also not a big fan of how we have to do API routes in next js.. I like the traditional 'express' way of doing things more  where you import eg. auth/authRoutes.js (where you have all the routes + all their middleware) and you just server.use(authRoutes)